cmake/helpers.cmake

```diff
@@ -6,6 +6,10 @@
 # helpers.cmake -- helper functions for top-level CMakeLists.txt
 #
```

Comment:
pls bump dates (in all files)

Reply:
Done.

Reply:
not done in the top-level CMakeLists

Reply:
Done: https://github.com/oneapi-src/unified-memory-framework/pull/150